src/test/groovy/org/gradle/profiler/ProfilerIntegrationTest.groovy

```diff
@@ -250,29 +250,111 @@ println "<daemon: " + gradle.services.get(org.gradle.internal.environment.Gradle
         assertBuildScanPublished()
     }
 
-    def "profiles build using Build Scans overridden version specified Gradle version and tasks"() {
+    def "uses Gradle Enterprise plugin version used by the build if present"() {
         given:
+        settingsFile.text = """
+plugins {
```

Comment:
💄 Indentation of the code code snippets here and in the rest of the tests.

I found that it does make a difference, when one needs to look at this tests at any future point and debug/fix them.

Reply:
I made indentation of ProfilerIntegrationTest consistent, but not for other classes :D.

Reply:
Thanks!